.github/ISSUE_TEMPLATE/new-feature.yaml

```
@@ -0,0 +1,44 @@
+name: Request documentation for an upcoming product change
+description: Describe an upcoming change to Astro or Astronomer Software that needs documentation
+assignees:
+  - jwitz
+  - kwcanuck
+labels: ["new-feature"]
+body:
+  - type: dropdown
+    id: impact
+    attributes:
+      label: What kind of change are you requesting? 
+      multiple: true
+      options:
+        - Setup steps for an existing feature or supported configuration
```

```suggestion
        - A procedure for an existing feature or supported configuration
```
If it already exists, why would a user need it documented again? Would this be better as:

_An update to an existing feature or supported configuration_

What the heck is a "supported configuration"?

---

This procedure they're requesting would not exist in docs yet. It might be something in the product that isn't yet documented. It could also be a way of setting up Astronomer that we endorse/ support but isn't yet documented yet, which is what I meant by "supported configuration". For example, someone might suggest "Best practices for managing worker queues". We have the core procedure documented for creating worker queues, but we don't have anything about how you would configure them to optimize performance.